spylib/oauth/types.py

```diff
@@ -0,0 +1,111 @@
+from typing import List
+
+from pydantic import BaseModel, validator
+
+
+def _parse_scope(scope: str) -> List[str]:
+    return scope.split(',')
+
+
+class OfflineTokenModel(BaseModel):
```

Comment:
I couldn't decide between the two actually. Settled on types because I was planning to add some non-model types as well. I don't have a strong feeling on this one

Reply:
> Do we want to merge the models from [admin_api](https://github.com/SatelCreative/spylib/blob/main/spylib/admin_api.py#L42-L63) in this PR as well?

Can we do that in a different PR? I'm not sure why there are oauth models in the admin api

Reply:
> I couldn't decide between the two actually. Settled on types because I was planning to add some non-model types as well. I don't have a strong feeling on this one



> I think I'd be more for `models` but not a deal breaker. Maybe @lishanl needs to be the tie breaker on naming! 😉

I am more for `models` in that case. 😄

Reply:
I'll change it